src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -41,6 +42,16 @@ def __init__(
         # set back to False if battery is back above warning level
         self.battery_warning_disabled = False
 
+        # True means the robot is receiving constant RobotStatus messages
+        # False means that the robot has received no RobotStatus for at least 1 tick
+        # used to grey out display when robot disconnects
+        self.is_connected = False
+
+        # True means the timer to grey oyt the UI has been triggered
+        # to prevent too many timers and UI flashing
+        self.to_be_disconnected = False
```

Comment:
```suggestion
        # True means the timer to grey out the UI has been triggered
        # to prevent too many timers and UI flashing
        self.to_be_disconnected = False
```